samples/CallingStateful/src/App.tsx

```diff
@@ -85,6 +88,7 @@ function App(): JSX.Element {
       } else if (event.removed.length > 0) {
         if (event.removed[0] === call) {
           setCall(undefined);
+          console.log(call.id, call.callEndReason);
```

Comment:
Remove console.log

Reply:
i thought this was here on purpose

Reply:
Since the app is able to handle multiple calls should we not in the sample include the call end reason?

Reply:
Since this is a sample app, I am okay with keeping this console log here for quicker problem solving